swupd/delta.go

```diff
@@ -35,8 +35,8 @@ type Delta struct {
 // CreateDeltas creates all delta files between the previous and current version of the
 // supplied manifest. Returns a list of deltas (which contains information about
 // individual delta errors). Returns error (and no deltas) if it can't assemble the delta
-// list.
-func CreateDeltas(manifest, statedir string, from, to uint32) ([]Delta, error) {
+// list. If number of workers is zero or less, 1 worker is used.
```

Comment:
Would be good to add to the description of the cmd line arg, so it's clear what default is.

Reply:
Note the default for the package is different than what we expose in the command line. Usually library shouldn't be making assumptions about how much resources to use, that is caller code job.